### PR TITLE
Add /sellers/:seller_id/campaigns

### PR DESCRIPTION
**Pull request by rjvani**

### CODE REVIEW

File: app/controllers/seller_campaigns_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+class SellerCampaignsController < ApplicationController
+  before_action :set_seller
+
+  # GET /sellers/:seller_id/campaigns
+  def index
+    @campaigns = if params[:active].present?
+                   valid_campaigns.order(:end_date).active(params[:active])
+                 else
+                   valid_campaigns.order(:end_date).all
```
Comment (justintmckibben) [inline app/controllers/seller_campaigns_controller.rb:11]: Do you need the all?

File: spec/requests/seller_campaigns_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      let(:seller_id) { @seller.seller_id }
+
+      context 'Without an active flag' do
+        before { get "/sellers/#{seller_id}/campaigns" }
+
+        it 'Returns all campaigns for the seller' do
+          expect(json).not_to be_empty
+          expect(json.size).to eq(2)
+
+          seller_ids = json.map{|campaign| campaign['seller_id'] }
```
Comment (justintmckibben) [inline spec/requests/seller_campaigns_request_spec.rb:40]: mind running rubocop -a?

File: spec/requests/seller_campaigns_request_spec.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'SellerCampaigns', type: :request do
+  before do
```
Comment (justintmckibben) [inline spec/requests/seller_campaigns_request_spec.rb:4]: can you add a invalid campaign?

File: app/controllers/seller_campaigns_controller.rb (hunk 1 above)
Comment (rjvani) [reply app/controllers/seller_campaigns_controller.rb]: Guess not ¯\_(ツ)_/¯

File: spec/requests/seller_campaigns_request_spec.rb (hunk 2 above)
Comment (rjvani) [reply spec/requests/seller_campaigns_request_spec.rb]: Done

File: spec/requests/seller_campaigns_request_spec.rb (hunk 3 above)
Comment (rjvani) [reply spec/requests/seller_campaigns_request_spec.rb:6]: Yep, added a test for `valid` as well

**Pull request merged**